server/src/main/webapp/WEB-INF/rails/webpack/views/components/environment_variables/index.tsx

```diff
@@ -20,20 +20,24 @@ import s from "underscore.string";
 import {ButtonIcon, Secondary} from "views/components/buttons";
 import {EnvironmentVariableWidget} from "./environment_variable_widget";
 
-interface GroupedEnvironmentVariablesAttrs {
+export interface GroupedEnvironmentVariablesAttrs {
   title: string;
   environmentVariables: EnvironmentVariables;
   onAdd: () => void;
   onRemove: (environmentVariable: EnvironmentVariable) => void;
 }
 
-class GroupedEnvironmentVariables extends MithrilComponent<GroupedEnvironmentVariablesAttrs> {
+export class GroupedEnvironmentVariables extends MithrilComponent<GroupedEnvironmentVariablesAttrs> {
+  renderEnvironmentVariableWidget(vnode: m.Vnode<GroupedEnvironmentVariablesAttrs>, environmentVariable: EnvironmentVariable) {
+    return <EnvironmentVariableWidget environmentVariable={environmentVariable} onRemove={vnode.attrs.onRemove}/>;
+  }
+
   view(vnode: m.Vnode<GroupedEnvironmentVariablesAttrs>): m.Children {
     return <div>
       <h4 data-test-id={`${s.slugify(vnode.attrs.title)}-title`}>{vnode.attrs.title}</h4>
       {
         vnode.attrs.environmentVariables.map((envVar) => {
-          return <EnvironmentVariableWidget environmentVariable={envVar} onRemove={vnode.attrs.onRemove}/>;
+          return this.renderEnvironmentVariableWidget(vnode, envVar);
         })
```

Comment:
The Closure is not required as there is only one statement. Can you pls convert it.